flink/v1.20/flink/src/main/java/org/apache/iceberg/flink/TypeToFlinkType.java

```diff
@@ -113,6 +113,15 @@ public LogicalType primitive(Type.PrimitiveType primitive) {
           // MICROS
           return new TimestampType(6);
         }
+      case TIMESTAMP_NANO:
+        Types.TimestampNanoType timestampNano = (Types.TimestampNanoType) primitive;
+        if (timestampNano.shouldAdjustToUTC()) {
+          // NANOS
```

Comment:
Why is this comment?

Reply:
to be consistent with current comments already present for `MICROS`, ie:
```java
 if (timestamp.shouldAdjustToUTC()) {
          // MICROS
          return new LocalZonedTimestampType(6);
        } else {
          // MICROS
          return new TimestampType(6);
        }
```